src/__tests__/pwnedPassword.test.ts

```diff
@@ -19,7 +19,7 @@ describe('pwnedPassword', () => {
     it('resolves to 0', () => {
       server.use(
         rest.get('*', (_, res, ctx) => {
-          return res(ctx.json(null));
+          return res(ctx.text(EXAMPLE_PASSWORD_HASHES));
```

Comment:
Was this change made because there should always be matching suffixes returned? Just curious.